kateryna_kravchuk/task_04/task_04_3.sh

```
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# create log file or clear it if existing
+> log.log
```

Shellcheck (0.10.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)